kore/app/exec/Main.hs

```diff
@@ -375,7 +381,11 @@ parserInfoModifiers =
 main :: IO ()
 main = do
     options <- mainGlobal parseKoreExecOptions parserInfoModifiers
-    Foldable.forM_ (localOptions options) mainWithOptions
+    Foldable.forM_ (localOptions options) $ \o ->
+        catch (mainWithOptions o) $
+            \(Goal.GoalException (e :: SomeException) p) -> do
+                renderResult o ("last configuration = " <> unparse p)
```

Comment:
I think here you don't want to print any additional text, as the KORE parser int the frontend would not be able to handle it.
Alternatively, consider making the additional text into a comment.